packages/app/src/cli/utilities/developer-platform-client/partners-client.ts

```diff
@@ -0,0 +1,44 @@
+import {DeveloperPlatformClient, Paginateable} from '../developer-platform-client.js'
+import {fetchPartnersSession, PartnersSession} from '../../../cli/services/context/partner-account-info.js'
+import {fetchAppDetailsFromApiKey, fetchOrganizations, fetchOrgAndApps} from '../../../cli/services/dev/fetch.js'
+import {MinimalOrganizationApp, Organization, OrganizationApp} from '../../models/organization.js'
+import {AbortError} from '@shopify/cli-kit/node/error'
+
+const resetHelpMessage = ['You can pass', {command: '--reset'}, 'to your command to reset your app configuration.']
+
+export class PartnersClient implements DeveloperPlatformClient {
+  private _session: PartnersSession | undefined
+
+  async session(): Promise<PartnersSession> {
+    if (!this._session) {
+      this._session = await fetchPartnersSession()
+    }
```

Comment:
It's probably because it's quite new. I think it's ok to have both expressions in the same codebase, but yeah, we can try to change a bunch of them later.